code/android-media-library/src/androidTest/java/com/adobe/marketing/mobile/media/MediaDBServiceTests.java

```diff
@@ -273,4 +275,26 @@ public void test_deleteAllHits_multipleSessionsPresent() {
             assertTrue(hits.isEmpty());
         }
     }
+
+    @Test
+    public void test_deleteDeprecatedDatabaseFile() throws IOException {
+        // Setup
+        Context context = ApplicationProvider.getApplicationContext();
+        final String DEPRECATED_2X_DB_FILE_NAME = "ADBMobileMedia.sqlite";
+        File oldDB = new File(context.getCacheDir(), DEPRECATED_2X_DB_FILE_NAME);
+        oldDB.createNewFile();
+
+        File newDB = context.getDatabasePath(MEDIA_DATABASE);
+        newDB.delete();
+
+        // Before upgrade
+        assertTrue(oldDB.exists());
+        assertFalse(newDB.exists());
+
+        MediaDBService dbService = new MediaDBServiceImpl();
```

Comment:
Should we also cover the flow where we don't have existing oldDb. Make sure everything works fine.

Reply:
The regular tests in the class already covers the scenario